.travis.yml

```
@@ -1,14 +1,15 @@
+dist: xenial
 language: python
 sudo: false
-python: 2.7
-env:
-    - TOX_ENV=py27
-    - TOX_ENV=py34
-    - TOX_ENV=flake8
 matrix:
-    allow_failures:
-        - env: TOX_ENV=py34
+  include:
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
```

So we can tell Travis to run both the 3.7 envs here and save a couple lines:
```suggestion
      env: TOXENV=py37,py37-flake8
```

---

Hmmm, although that saves a couple lines here, the test will not show up as a separate environment in Travis. So up to you whether you think it's worth doing it that way or not.

---

I think I'll leave it the way it is now.